extension.js

```diff
@@ -736,10 +736,8 @@ function activate (context) {
 		"dispose": () => suppressLint(throttle.document)
 	});
 
-	// Request (deferred) lint of active document
-	if (vscode.window.activeTextEditor) {
-		requestLint(vscode.window.activeTextEditor.document);
-	}
+	// Lint all visible documents
+	setTimeout(() => cleanLintVisibleFiles(), throttleDuration);
```

Comment:
Do we need a function that calls `cLVF`, or can we pass it directly by name?

Reply:
Done in ba0da26. 

It would be great to have a global ESLint rule similar to Lodash's [`no-extraneous-function-wrapping`](https://github.com/jfmengels/eslint-plugin-lodash-fp/blob/master/docs/rules/no-extraneous-function-wrapping.md) available.